src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AttendanceController.java

```diff
@@ -40,7 +43,19 @@ public void postAttendance(
             @RequestBody
             @Valid final CreateAttendanceDto attendance) {
 
-        this.attendanceService.createOffenderAttendance(attendance);
+        this.attendanceService.createAttendance(attendance);
+    }
+
+    @PutMapping(path = "/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    public ResponseEntity<Object> putAttendance(@PathVariable("id") long id, @RequestBody @Valid final UpdateAttendanceDto attendance) {
+        try {
+            this.attendanceService.updateAttendance(id, attendance);
+        } catch (AttendanceNotFound e) {
+            return ResponseEntity.status(404).body(null);
+        }
+
+        return ResponseEntity.ok(null);
```

Comment:
should really be a `ResponseEntity.noContent().build()` if you're not returning anything i.e. 204 instead of 200